test/integration/cloud_event.ts

```diff
@@ -53,33 +47,33 @@ describe('CloudEvent Function', () => {
     {
       name: 'CloudEvents v1.0 structured content request',
       headers: {'Content-Type': 'application/cloudevents+json'},
-      body: TEST_CLOUD_EVENT,
-      expectedCloudEvent: TEST_CLOUD_EVENT,
+      body: testCloudEvent(),
+      expectedCloudEvent: testCloudEvent(),
     },
     {
       name: 'CloudEvents v1.0 structured content request',
       headers: {},
-      body: TEST_CLOUD_EVENT,
+      body: testCloudEvent(),
       expectedCloudEvent: {
-        ...TEST_CLOUD_EVENT,
+        ...testCloudEvent(),
       },
     },
     {
       name: 'CloudEvents v1.0 binary content request',
       headers: {
         'Content-Type': 'application/json',
-        'ce-specversion': TEST_CLOUD_EVENT.specversion,
-        'ce-type': TEST_CLOUD_EVENT.type,
-        'ce-source': TEST_CLOUD_EVENT.source,
-        'ce-subject': TEST_CLOUD_EVENT.subject,
-        'ce-id': TEST_CLOUD_EVENT.id,
-        'ce-time': TEST_CLOUD_EVENT.time,
-        'ce-datacontenttype': TEST_CLOUD_EVENT.datacontenttype,
+        'ce-specversion': testCloudEvent().specversion,
+        'ce-type': testCloudEvent().type,
+        'ce-source': testCloudEvent().source,
+        'ce-subject': 'test-subject',
+        'ce-id': testCloudEvent().id,
+        'ce-time': testCloudEvent().time,
+        'ce-datacontenttype': testCloudEvent().datacontenttype,
```

Comment:
I believe it's pretty odd to call a function 7 times to get the properties of the returned object. We're using the same CloudEvent object.

In this instance, we're getting 7 different objects, then just getting one property from them.

Can we keep using the static CloudEvent, especially if it is frozen?

Reply:
reverted this as I removed `testCloudEvent`